CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.8
+
+* Require the latest version of `package:http_parser`.
```

Comment:
I was hoping to help folks who are just running `pub get` – they may still get the old thing.

Also, I was planning to push a few more commits before I wanted to release – agree w/ the eventual version, though.